_data/sites.json

```
@@ -10534,6 +10534,16 @@
         ]
     },
 
+      {
+        "name": "Lush",
+        "url": "https://www.lush.com/us/en_us/faq/usa-contact-us",
+        "notes": "Conctact customer service and request account deletion",
+        "difficulty": "hard",
+        "domains": [
+            "lush.com"
```

There are more domains:

```suggestion
            "lush.com",
            "forms.lush.com",
            "weare.lush.com"
```